standard/template_engine.lua

```diff
@@ -0,0 +1,149 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:TemplateEngine
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Table = require('Module:Table')
+
+---Return true if the input is a table in array format
+---@param array any
+---@return boolean
+local function isArray(array)
+	if type(array) ~= 'table' then
+		return false
+	end
+	return Table.size(array) == #array
+end
```

Comment:
is this maybe something we would want in `Module:Array`?

Reply:
#2292